package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "devfile-landing-page",
+  "author": "Braden Wilson",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "next",
+    "build": "next build && next export",
+    "start": "next start",
+    "start-build": "yarn build & yarn start",
+    "clean": "rm -rf .next/ out/ node_modules/",
+    "cypress:start": "yarn cypress open",
+    "cypress:start-build": "concurrently --names 'CYPRESS,SERVER' --prefix-colors 'yellow,blue' \"yarn cypress:start\" \"yarn start-build\"",
+    "typedoc:build": "node_modules/.bin/typedoc --tsconfig .",
+    "typedoc:start": "npx serve docs",
+    "jest:test": "jest --watchAll --verbose",
+    "test": "cypress run",
```

Comment:
So, do you think we need to test this repo? Probably makes sense? Just to see if a page opens when you click it

Reply:
cypress incl